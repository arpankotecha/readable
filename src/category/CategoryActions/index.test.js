import * as actions from './index'

it('creates a new addCategory action', () => {
  const addCategory = actions.addCategory("newCategory")
  expect(addCategory).toMatchSnapshot()
})

