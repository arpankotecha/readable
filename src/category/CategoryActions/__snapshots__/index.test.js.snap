// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a new addCategory action 1`] = `
Object {
  "category": "newCategory",
  "type": "ADD_CATEGORY",
}
`;
